### ISSUE TITLE
Security vulnerability in mpath 0.8.3

### ISSUE THREAD

**Issue by uzair004** [Automattic/mongoose#10774]
https://github.com/Automattic/mongoose/blob/d486ca118f0aa14d17f543de300c98666639a733/package.json#L27 

mpath version `0.8.3` has security vulnerability, still being used in `mongoose 5.13` and prior version.
Please consider bumping mpath to `0.8.4`  or we should manually upgrade mpath to 0.8.4. Will it break anything ?

**Comment by sean-daley** [comment]
5.13.9 was released on September 6. This included an update to mpath to 0.8.4.

```
5.13.9 / 2021-09-06
fix(populate): avoid setting empty array on lean document when populate result is undefined #10599
fix(document): make depopulate() handle populated paths underneath document arrays #10592
fix: peg @types/bson version to 1.x || 4.0.x to avoid stubbed 4.2.x release #10678
fix(index.d.ts): simplify UpdateQuery to avoid "excessively deep and possibly infinite" errors with extends Document and any #10647
fix(index.d.ts): allow specifying weights as an IndexOption #10586
fix: upgrade to mpath v0.8.4 re: security issue #10683
```

**Comment by uzair004** [comment]
> 5.13.9 was released on September 6. This included an update to mpath to 0.8.4.
> 
> ```
> 5.13.9 / 2021-09-06
> fix(populate): avoid setting empty array on lean document when populate result is undefined #10599
> fix(document): make depopulate() handle populated paths underneath document arrays #10592
> fix: peg @types/bson version to 1.x || 4.0.x to avoid stubbed 4.2.x release #10678
> fix(index.d.ts): simplify UpdateQuery to avoid "excessively deep and possibly infinite" errors with extends Document and any #10647
> fix(index.d.ts): allow specifying weights as an IndexOption #10586
> fix: upgrade to mpath v0.8.4 re: security issue #10683
> ```

@sean-daley Thanks for comment, Issue closed.

Is there any breaking changes b/w mongoose `5.12.7` and `5.13.9` ?
I wanna update but not sure If it will break my code.

**Comment by vkarpov15** [comment]
@uzair004 no intentional breaking changes between v5.12.7 and v5.13.9
